python/python/tests/test_vector_index.py

```diff
@@ -78,7 +78,8 @@ def run(ds, q=None, assert_func=None):
             expected_columns.extend(ds.schema.names)
         else:
             expected_columns.extend(columns)
-        for c in ["vector", "_distance"]:
+        # TODO: _distance shouldn't be returned by default either
+        for c in ["_distance"]:
             if c not in expected_columns:
                 expected_columns.append(c)
```

Comment:
This feels a little silly as a for loop now. Maybe:

```suggestion
        if "_distance" not in expected_columns:
            expected_columns.append(c)
```